lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -133,6 +136,30 @@ def test_credit_requirements_not_eligible(self):
         )
         self.assertContains(response, "Verification Failed")
 
+    @ddt.data(CourseMode.HONOR, CourseMode.AUDIT, CourseMode.PROFESSIONAL)
+    def test_credit_requirements_non_credit_enrollment(self, enrollment_mode):
+        # Test the progress table is not displayed to the non credit students.
+        self.enrollment.mode = enrollment_mode
+        self.enrollment.save()  # pylint: disable=no-member
+
+        # Check the progress page display
+        response = self._get_progress_page()
+        self.assertNotContains(response, "<section class=\"credit-eligibility\">")
```

Comment:
Make `"<section class=\"credit-eligibility\">"` a class variable, because if the markup ever changes, then `test_credit_requirements_credit_verified_enrollment` will fail, but this test will not. We want to make sure that this test passes when test_credit_requirements_credit_verified_enrollment passes